src/engine.rs

```diff
@@ -873,6 +899,7 @@ impl<'a, 'b> Cpu<'a, 'b> {
             x if x == self.engine.config.address.uart => {
                 let mut buffer = self.engine.putchar_buffer.lock().unwrap();
                 let buffer = buffer.entry(self.hartid).or_default();
+                debug!("UART Store: addr 0x{:x} value 0x{:x}", addr, value);
```

Comment:
I think we should remove this one again, or make it a `trace` as well to not pollute the debug log too much.

Reply:
Addressed in [this commit](https://github.com/pulp-platform/banshee/pull/18/commits/95619028583909da22c571e6fdbf3cd400d35545).